website/app/components/doc/list-container.hbs

```diff
@@ -0,0 +1,3 @@
+<ul ...attributes>
```

Comment:
what do you perceive the value is of this vs using a `<ul>` directly in the markdown?

Reply:
the ul in the markdown doesn't render. If there's some trick to it that I just don't know, I'm fine with learning that trick instead. :)

Reply:
maybe can pair? i'm able to add arbitrary html to md files but i may be missing something about the usecase here.

Reply:
ahh, i see what you mean now. because it's _inside_ a handlebars codeblock you can't add arbitrary html